jobs/rocoto/ocnpost.sh

```diff
@@ -153,7 +150,7 @@ done
 # Restore CDATE to what is expected
 export CDATE=$IDATE
 echo $pwd
-$NMV ocn_ice*.grb2 $COMOUT
+$NMV ocn_ice*.grb2 $COMOUT/ocean/
 status=$?
 [[ $status -ne 0 ]] && exit $status
```

Comment:
we are checking on every CP and MV now?

Reply:
I leave it to you.  I would suggest removing this instance of status check for move.
OTOH, would creating soft-links help "making these files available faster".
Probably can defer that for a later PR.

Reply:
Now that there is proper exit code checking for all the ncks calls, I'm going to leave this so everything is being checked.